Source/WebCore/page/LocalFrameViewLayoutContext.h

```diff
@@ -93,6 +93,9 @@ class LocalFrameViewLayoutContext {
     bool isInRenderTreeLayout() const { return layoutPhase() == LayoutPhase::InRenderTreeLayout; }
     bool inPaintableState() const { return layoutPhase() != LayoutPhase::InRenderTreeLayout && layoutPhase() != LayoutPhase::InViewSizeAdjust && (layoutPhase() != LayoutPhase::InPostLayout || inAsynchronousTasks()); }
 
+    bool isSkippedContentLayoutForced() const { return m_isSkippedContentLayoutForced; }
+    void forceSkippedContentLayout(bool f) { m_isSkippedContentLayoutForced = f; }
+
```

Comment:
These should probably be on `RenderLayoutState`

Reply:
I did not see how that would work but have an idea now, will try to implement it tomorrow.